lib/transactional.pm

```diff
@@ -147,7 +148,27 @@ sub check_reboot_changes {
     process_reboot(trigger => 1) if $change_happened;
 }
 
-
+# Check slem target version after migration
+sub check_slem_target_version {
```

Comment:
there are two product names, SL Micro and SLEM, modifying zypper_migration.pm you need verification run for both and not include 'slem' in the name. You don't need to comment either.

Reply:
Updated.